pkg/redis/generate_scripts.go

```diff
@@ -59,7 +59,7 @@ var (
 )`))
 
 func isCommentLine(s string) bool {
```

Comment:
This generator may be removed once we move to `go` `1.16` via file embedding. Consider creating an issue about it.

References https://github.com/golang/go/issues/41191